core/src/test/java/tachyon/conf/TachyonConfTest.java

```diff
@@ -227,6 +229,12 @@ public void testUserDefault() {
 
     longValue = sDefaultTachyonConf.getBytes(Constants.USER_REMOTE_READ_BUFFER_SIZE_BYTE, 0);
     Assert.assertTrue(longValue == Constants.MB);
+
+    Class<? extends RemoteBlockReader> reader =
+        sDefaultTachyonConf.getClass(Constants.USER_REMOTE_BLOCK_READER,
+            Constants.USER_REMOTE_BLOCK_READER_CLASS);
```

Comment:
When you add the USER_REMOTE_BLOCK_READER_CLASS to tachyon-default.properties files, the test should use null here to make sure the TachyonConf load it properly.